src/components/__tests__/__snapshots__/CvssNISTForm.spec.ts.snap

```diff
@@ -17,9 +17,7 @@ exports[`cvssNISTForm > renders a modal when clicking the button 1`] = `
     <div data-v-296c460d="" class="modal fade show" style="display: block;" aria-hidden="false" tabindex="-1" aria-labelledby="modalTitle" role="dialog">
       <div data-v-296c460d="" class="modal-dialog">
         <div data-v-296c460d="" class="modal-content">
-          <div data-v-296c460d="" class="modal-header">
-            <h1 data-v-296c460d="" class="modal-title fs-5"> Email NIST </h1><button data-v-296c460d="" type="button" class="btn-close" aria-label="Close"></button>
-          </div>
+          <!--v-if-->
```

Comment:
Seems like we lost this since the new `v-if="$slots.header"` in the Modal component

Reply:
Thanks for catching that, will fix it.